app/views/api/v1/conversations/_info.json.jbuilder

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+json.sender(conversation.sender.name)
```

Comment:
don't you need also the id for these 3 objects?

Reply:
I will add this info to the response! @JuanfraM